plugins/usi-dock/README.md

```diff
@@ -0,0 +1,37 @@
+# USI Dock
+
+## Introduction
+
+This plugin uses the MCU to write all the dock firmware components. Interestingly, the MCU version
+is provided by the DMC bcdDevice.
+
+This plugin supports the following protocol ID:
+
+* com.usi.dock
+
+## GUID Generation
+
+These devices use the standard USB DeviceInstanceId values, e.g.
+
+* `USB\VID_17EF&PID_7226&REV_0001`
+* `USB\VID_17EF&PID_7226`
+* `USB\VID_17EF`
+
+Additionally, some extra "component ID" instance IDs are added.
+
+* `USB\VID_17EF&PID_7226&CID_TBT4`
```

Comment:
I think it should be providing exact same component ID for TBT NVM so that thunderbolt plugin or this one can handle it (but set priorities accordingly for preferred one)

Reply:
This plugin handles MCU instead of TBT devices. The component ID is used for display only.

Reply:
If this plugin is supposed to entirely handle TBT - my opinion is the TBT plugin needs to be disabled for any devices that are assumed to be controlled here when they're registered.  Perhaps model it off of this:
https://github.com/fwupd/fwupd/blob/main/plugins/dell-dock/fu-plugin-dell-dock.c#L221